### ISSUE TITLE
NextDenovo Parent Job Failed, No Error, Subtasks still running

### ISSUE THREAD

**Issue by harveyzac** [Nextomics/NextDenovo#172]
Hello,

I am using NextDenovo to assemble a genome from a PromethIon run. It was running for about a week, but then the parent job that I submitted failed, with the error message:

[35943 ERROR] 2023-04-19 18:52:46 seed_cns failed: please check the following logs:
[35943 ERROR] 2023-04-19 18:52:46 /02.cns_align/01.seed_cns.sh.work/seed_cns1/nextDenovo.sh.e
[35943 ERROR] 2023-04-19 18:52:46 /02.cns_align/01.seed_cns.sh.work/seed_cns2/nextDenovo.sh.e
[35943 ERROR] 2023-04-19 18:52:46 /02.cns_align/01.seed_cns.sh.work/seed_cns3/nextDenovo.sh.e
[35943 ERROR] 2023-04-19 18:52:46 /02.cns_align/01.seed_cns.sh.work/seed_cns4/nextDenovo.sh.e
[35943 ERROR] 2023-04-19 18:52:46 /02.cns_align/01.seed_cns.sh.work/seed_cns5/nextDenovo.sh.e

However, I checked these log files, and there are no errors there. The last line from one of them is:
[38173 INFO] 2023-04-11 17:15:46 Start a cns worker in 38173 from parent 37241

And the relevant jobs are still running. 

Is it possible to just wait for them to complete and then restart the job at the next step?

**Comment by moold** [comment]
No, try to kill all running subtasks and rerun the main task, it will continue to run unfinshed tasks.
